install-hooks.sh

```
@@ -0,0 +1,458 @@
+#!/usr/bin/env bash
```

Shellcheck (0.11.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)